transactions/transactions_service.go

```diff
@@ -607,14 +620,19 @@ func (svc *transactionsService) ConsumeEvent(ctx context.Context, event *events.
 
 			if result.RowsAffected == 0 {
 				// Note: brand new payments cannot be associated with an app
-				var metadata string
+				var metadataBytes []byte
```

Comment:
I actually don't think we should do this on `nwc_payment_sent` because we already save the metadata when we initiate the payment

Reply:
Not that important but what if payments happen outside (they don't have metadata since it is a payment, atleast in LND but still I thought it was good to have if we are anyways adding metadata)

Also we should DRY up the "brand new" lnClient.Transaction processing because it is almost the same in both sent/received

Reply:
I decided to remove it for now - we need to confirm if we want to add this support and if so, also import transactions from LND (all this is a lot of extra work and code to do properly)